tests/end-to-end/log-manager/setup_suite.bash

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+setup_suite() {
+  export CK8S_AUTO_APPROVE="true"
+
+  load "../../bats.lib.bash"
+
+  with_kubeconfig sc
+  with_namespace fluentd-system
+
+  # Pause logs being flushed to object storage
+  fluentd_aggregator_replicas="$(kubectl -n "${NAMESPACE}" get sts fluentd-aggregator -o jsonpath='{.spec.replicas}')"
```

Comment:
Readability nit, no need to save on typing in scripts 😉 (two more `sts`, tho having at least one instance expanded helps remember what that short form meant)

```suggestion
  fluentd_aggregator_replicas="$(kubectl -n "${NAMESPACE}" get statefulset fluentd-aggregator -o jsonpath='{.spec.replicas}')"
```

Maybe the `-o` too if we're being especially pedantic 😁

Reply:
Yes, my bad, old habits die hard.